permissions/exocompute/8.json

```diff
@@ -0,0 +1,361 @@
+{
+    "Statement": [
+      {
+        "Sid": "ExocomputeDeleteClusterSid",
+        "Effect": "Allow",
+        "Action": [
+          {
+            "Permission": "eks:DeleteCluster",
+            "UseCases": [
+              "Deleting an EKS cluster."
+            ]
+          },
+          {
+            "Permission": "autoscaling:DeleteAutoScalingGroup",
+            "UseCases": [
+              "Deleting an auto-scaling group launched for worker nodes."
+            ]
+          }
+        ],
+        "Resource": [
+          "*"
+        ],
+        "Condition": {
+          "StringEquals": {
+            "aws:ResourceTag/rk_component": "Cloud Native Protection"
+          }
+        }
+      },
+      {
+        "Sid": "ExocomputeDeleteLaunchConfigSid",
+        "Effect": "Allow",
+        "Action": [
+          {
+            "Permission": "autoscaling:DeleteLaunchConfiguration",
+            "UseCases": [
+              "Deleting an auto-scaling group launched for worker nodes."
+            ]
+          }
+        ],
+        "Resource": [
+          "arn:*:autoscaling:*:*:launchConfiguration:*:launchConfigurationName/Rubrik-*"
+        ]
+      },
+      {
+        "Sid": "ExocomputeDeleteLaunchTemplateSid",
+        "Effect": "Allow",
+        "Action": [
+          {
+            "Permission": "eks:DeleteLaunchTemplate",
+            "UseCases": [
+              "Deleting a launch template created for worker nodes."
+            ]
+          }
+        ],
+        "Resource": [
+          "*"
```

Comment:
Do we want this permission on all the resource? Or can we reduce the scope by adding some constraints.

Reply:
we do have. added constraint on delete by using tags. (resource arn only allowed setting filter by launch template id, which we did not have any control over while creation, hence using tags)